src/initial_conditions/atmos_state.jl

```diff
@@ -140,10 +151,45 @@ function turbconv_center_variables(ls, turbconv_model::PrognosticEDMFX, gs_vars)
     sgsʲs = ntuple(_ -> (; ρa = ρa, mse = mse, q_tot = q_tot), Val(n))
     return (; sgs⁰, sgsʲs)
 end
+function turbconv_center_variables(
```

Comment:
I think this function can be combined with the previous one using the same pattern in `grid_scale_center_variables`. But I'm fine with keep it as it is and unify them later. Maybe add a todo?